rpcs3/Emu/Cell/SPUThread.h

```diff
@@ -678,6 +679,12 @@ class spu_thread : public cpu_thread
 
 	void fast_call(u32 ls_addr);
 
+	void wakeup_delay(u32 div = 1) const
```

Comment:
Can you implement this in the cpp in order to move the include?

Reply:
I dont recommend doing this as this nearly kills any chance for inling.